src/handler.py

```diff
@@ -96,10 +100,25 @@ def start_observations_db(event, context):
         boto3.client('rds').start_db_instance(DBInstanceIdentifier='observations-qa')
 
 
+def control_db_utilization(event, context):
+    """
+    Right now we are only listening for the error handler alarm, because it
+    is the last alarm to get triggered when we are going into a death spiral.
+    :param event:
+    :param context:
+    :return:
+    """
+    logger.info(event)
+    alarm_state = event["detail"]["state"]["value"]
+    #if alarm_state == "ALARM":
```

Comment:
I'm a bit confused. I don't see anything in these changes that explicitly use the errorHandler's alarm... :thinking:

Reply:
Fixed.